templates/engage/casestudy-itstratagen.html

```diff
@@ -0,0 +1,30 @@
+{% extends "engage/base_engage.html" %}
+
+{% block title %}ITstrategen keeps legacy applications secure with Extended Security Maintenance{% endblock %}
+
+{% block meta_description %}Extended Security Maintenance extends official support for the operating system, guaranteeing the continued security of ITstrategen’s servers and saving its clients from costly application updates.{% endblock %}
+
+{% block meta_copydoc %}https://app-sjg.marketo.com/#LP5353A1LA1{% endblock meta_copydoc %}
+
+{% block content %}
+
+{% include "engage/shared/_header.html" with title="ITstrategen keeps legacy applications secure with Extended Security Maintenance" image="7005c900-pictogram_article01.png" url="#the-form" cta="Download the case study" %}
+
+<section class="p-strip is-shallow">
+  <div class="row">
+    <div class="col-7">
+        <p>Extended Security Maintenance (ESM) extends official support for the operating system, guaranteeing the continued security of ITstrategen’s servers and saving its clients from costly application updates.</p>
```

Comment:
please use 2 spaces indentation instead of 4 as this is a Canonical best practice https://canonical-web-and-design.github.io/practices/coding/html.html